web-platform/website/src/components/pages/privacy-policy/StandardPrivacyPolicySection.tsx

```diff
@@ -16,24 +16,31 @@ export function StandardPrivacyPolicySection({
   tagline,
   sections,
 }: StandardPrivacyPolicySectionProps) {
+  const isExtensionView = localStorage.getItem("isExtensionView");
```

Comment:
We should use the v2 flag here? Or create a new flag for this? We should not use local storage because it will propagate to end users which we don't want.

Reply:
Yes, this was meant as a temporary solution. I originally used the v2 flag here. I'll revert to what I had before.